Symfony/CS/Fixer/Symfony/PhpdocInlineTagFixer.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Symfony;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * Fix inline tags and make inheritdoc tag always inline.
+ */
+final class PhpdocInlineTagFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function isCandidate(Tokens $tokens)
+    {
+        return $tokens->isTokenKindFound(T_DOC_COMMENT);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, Tokens $tokens)
+    {
+        foreach ($tokens as $token) {
+            if (!$token->isGivenKind(T_DOC_COMMENT)) {
+                continue;
+            }
+
+            $content = $token->getContent();
+
+            // Move `@` inside tag, for example @{tag} -> {@tag}, replace multiple '{'/'}',
+            // remove spaces between '{' and '@', remove 's' at the end of tag word
+            $content = preg_replace(
+                '#(@+[{]+|[{]+[ \t]*@+)[ \t]*(example|id|internal|inheritdoc|link|source|toc|tutorial)[s]*([^}]*)([}]*)#i',
```

Comment:
I remember that `\h` may tricky with some UTF chars stored on 2 bytes in some languages like in Japanese